zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepoSync.java

```diff
@@ -137,17 +137,14 @@ public void remove(String noteId) throws IOException {
    * copy new/updated notes from source to destination storage
    * @throws IOException
    */
-  void sync(int sourceRepoIndex, int destRepoIndex) throws IOException {
+  public void sync(int sourceRepoIndex, int destRepoIndex) throws IOException {    
```

Comment:
I think we don't need to expose this function as public api yet, the reason for declaring it as a `private-package` was to call it only from `sync()`  as well as for tests. Changing it to public may become reasonable when we start to support more than 2 backend repos simultaneously (surely logic should be changed as well).